taskt/Core/Automation/Commands/Dictionary Commands/AddDictionaryItemCommand.cs

```diff
@@ -1,91 +1,71 @@
 ﻿using System;
-using System.Xml.Serialization;
-using System.Data;
 using System.Collections.Generic;
-using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
-using System.Drawing;
+using System.Data;
 using System.Linq;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Dictionary Commands")]
-    [Attributes.ClassAttributes.Description("This command Adds a key and value to a existing Dictionary")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to add to a dictionary")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Dictionary Commands")]
+    [Description("This command allows user to add an item (key and value pair) to a Dictionary.")]
     public class AddDictionaryItemCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Indicate Dictionary Name")]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate a Dictionary to add to")]
-        [Attributes.PropertyAttributes.SampleUsage("vMyDictionary")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Dictionary Name")]
+        [InputSpecification("Select the dictionary variable to add item to.")]
+        [SampleUsage("{vMyDictionary}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_DictionaryName { get; set; }
 
         [XmlElement]
-        [Attributes.PropertyAttributes.PropertyDescription("Define Keys and Values")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the Keys and Values required for your dictionary")]
-        [Attributes.PropertyAttributes.SampleUsage("")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Keys and Values")]
+        [InputSpecification("Enter Keys and Values required for the dictionary.")]
+        [SampleUsage("")]
```

Comment:
This needs a sample. For datatable inputs, I've been using this format: "[ FirstName | John ] || [{vKey} | {vValue} ]". Which reminds me that I need to add samples to the API datatable inputs.

Reply:
done